### ISSUE TITLE
Condition adjustment

### ISSUE THREAD

**Issue by Seesam24601** [Seesam24601/TAMMI#17]
Create an additional sample function to show another method of determining which assets are due for replacement that is based on asset condition instead of age.

This may need to be coupled with an additional function to determine how asset condition changes over time.

**Comment by Seesam24601** [comment]
Blocked by #1

**Comment by Seesam24601** [comment]
This should be able to be implemented easily by the end-user by adding a column to the `assets` table and then updating user-suppliable functions accordingly. Importantly, creation of the annual_adjustment user-suppliable function in #41 makes this relatively easy to implement.

This is not necessary for completing the core of the model or for a reasonable demonstration. Thus, I am closing this issue.

Providing built-in support for features like this may be re-added in a later phase of the project.
